client/app/scripts/superdesk-publish/filters/view/production-test-view.html

```diff
@@ -0,0 +1,51 @@
+<section class="main-section.filter">
+    <div class="preview-layout" ng-class="{closed: !selected.preview}">
+        <div class="list-pane">
+            <header>
+                <div class="sortbar pull-left">
+                    <span ng-repeat="type in resultType">
+                        <input type="radio" ng-click="fetchResults()" name="type" value="{{type.value}}" data-ng-model="model.selectedType"> {{type.id}}
```

Comment:
`type.value` does probably not change, and we can thus use one-time binding for it?